src/renderer_vk.cpp

```diff
@@ -2698,9 +2699,9 @@ VK_IMPORT_DEVICE
 
 			VkViewport vp;
 			vp.x        = 0;
-			vp.y        = 0;
+			vp.y        = (float)height;
```

Comment:
Reading extension here:
https://github.com/KhronosGroup/Vulkan-Docs/blob/master/appendices/VK_KHR_maintenance1.txt

Is this actually correct? It looks confusing...

Reply:
Negative height leads to (x,y) being interpreted as the lower-left instead of upper-left corner, so that's the correct way of doing it. Quote from the [spec](https://www.khronos.org/registry/vulkan/specs/1.0-extensions/html/chap25.html#VkViewport):
> The application can specify a negative term for height, which has the effect of negating the y coordinate in clip space before performing the transform. When using a negative height, the application should also adjust the y value to point to the lower left corner of the viewport instead of the upper left corner.

Reply:
Yep, could have mentioned my references
My main reference was: https://www.saschawillems.de/blog/2019/03/29/flipping-the-vulkan-viewport/
It works like a charm